Floofbot/Migrations/FloofDataContextModelSnapshot.cs

```diff
@@ -18,8 +18,12 @@ protected override void BuildModel(ModelBuilder modelBuilder)
 
             modelBuilder.Entity("Floofbot.Services.Repository.Models.LogConfig", b =>
                 {
+                    b.Property<ulong>("ServerId")
```

Comment:
These parts aren't reflected in the migration, but the bot still runs after?
If so, are we essentially manually modifying the context to reflect the DB, but not recording it in a migration?
If both are true then I think it is fine for now, as we should be migrating off of SQLite anyway later on.

Reply:
both are true, but to be honest we will be generating a new migration for when we move onto mysql